app/routes/people/person/personal-information.js

```diff
@@ -11,11 +17,99 @@ export default class PeoplePersonPersonalInformationRoute extends Route {
       include: [
         'mandatories.mandate.governing-body.is-time-specialization-of.administrative-unit.classification',
         'mandatories.contacts',
+        'agents-in-position',
       ].join(),
     });
+    const contacts = [];
+    const m = await person.mandatories;
+    console.log(m);
+    const mandatories = person.mandatories.filter(
+      (m) => m != null // hack because otherwise mandatories is not an array
+    );
+
+    const ministers = person.agentsInPosition.filter(
+      (m) => m != null // same
+    );
+
+    for (let mandatory of mandatories) {
+      const mandate = await mandatory.mandate;
+      if (!this.isActivePosition(mandate.endDate)) {
+        break;
+      }
+      const role = await mandate.roleBoard;
+      const governingBody = await mandate.governingBody;
+      const isTimeSpecializationOf = await governingBody.isTimeSpecializationOf;
+      const administrativeUnit =
+        await isTimeSpecializationOf.administrativeUnit;
+      const mContacts = await mandatory.contacts;
+      const primaryContact = findPrimaryContact(mContacts);
+      const secondaryContact = findSecondaryContact(mContacts);
+      contacts.push({
+        title: `${role.label}, ${administrativeUnit.name}`,
+        role: role.label,
+        type: 'mandatory',
+        id: mandatory.id,
+        startDate: mandate.startDate,
+        endDate: mandate.endDate,
+        unitId: administrativeUnit.id,
+        unitName: administrativeUnit.name,
```

Comment:
Why not pass the full administrative unit record instead? It seems weird to split this up in 2 props?